etc/ipmipower.dev

```diff
@@ -1,11 +1,21 @@
 # 
 # $Id$
 #
-# This device configuration supports power control via the 
+# This device configuration supports power control via the FreeIPMI
 # 'ipmipower' utility.
 #
-# See powerman-devices(7) for more info on running ipmipower under
-# powerman.
+# Powerman.conf will typically be setup like this:
+#
+#   include "/etc/powerman/ipmipower.dev"
+#   device "ipmipower1" "ipmipower" "/usr/sbin/ipmipower -h pcorona[1-64] |&"
+#   node "node1" "ipmipower1" "pnode1"
+#
+# The username and password for ipmipower may be configured on the command line
+# above or (recommended) via the FreeIPMI configuration file.
+#
+# Although not required, it is recommended ipmipower be configured
+# with the --wait-until-on and/or --wait-until-off options, so that
+# IPMI will emulate more traditional plug based power control.
```

Comment:
I would just add those options to the example and drop this paragraph.  The async behavior can lead to nasty surprises for users when power control is scripted e.g. for HA fail-over.